ssh/data/run.sh

```diff
@@ -47,8 +47,18 @@ touch /data/.bash_history
 chmod 600 /data/.bash_history
 ln -s -f /data/.bash_history /root/.bash_history
 
+# Persist .bash_profile by redirecting .bash_profile to /data
+touch /data/.bash_profile
+chmod 600 /data/.bash_profile
+ln -s -f /data/.bash_profile /root/.bash_profile
+
```

Comment:
More like:
```
if bashio::file.exists /data/.bash_profile; then
   sed -i ... /data/.bash_profile
else
  echo "export ..." > /data/.bash_profile
  chmod 600 /data/.bash_profile
fi
ln -s -f /data/.bash_profile /root/.bash_profile
```

Reply:
I've updated the PR to include binary logic on if the file exists to either create or replace the token if necessary.